packages/dataviews/src/components/dataviews-filters/search-widget.tsx

```diff
@@ -107,19 +101,20 @@ function ListBox( { view, filter, onChangeView }: SearchWidgetProps ) {
 				__( 'List of: %1$s' ),
 				filter.name
 			) }
+			// @ts-expect-error check if this actually works, since `onFocusVisible`
+			// only works when `focusable` is `true`.
 			onFocusVisible={ () => {
 				if ( ! compositeStore.getState().activeId ) {
 					compositeStore.move( compositeStore.first() );
 				}
 			} }
```

Comment:
Hey @ntsekouras could help me understand if we can actually remove the `onFocusVisible` callback here? That's because in theory, from what I understand from [the docs](https://ariakit.org/reference/composite#onfocusvisible), it shouldn't work without the `focusable` prop.

Does it currently work? If not, should we remove it altogether, or try to make it work by also passing the `focusable` prop?

Thank you!

Reply:
From what reading I've done, I think adding `focusable` should do it. It doesn't seem to work in `trunk` for me in my testing right now.

Reply:
Actually, it works as expected, both in trunk and here:

https://github.com/user-attachments/assets/07b735ae-0167-4935-8d42-8a75d4342a5c

so we should be good 👍

Reply:
Sorry for the late reply. It seems you figured it out already 😄 . The conditional rendering of different and what designers wanted led to this and similar nuances like [this one](https://github.com/WordPress/gutenberg/blob/trunk/packages/dataviews/src/components/dataviews-filters/search-widget.tsx#L90).